aasemble/django/apps/main/context_processors.py

```diff
@@ -0,0 +1,11 @@
+from test_project import settings
+
+
+def internal_name_change(request):
+    try:
+        title = settings.AASEMBLE_OVERRIDE_NAME
+    except AttributeError:
+        title = 'aaSemble'
```

Comment:
`title = getattr(settings, 'AASEMBLE_OVERRIDE_NAME', 'aaSemble')` should do the trick. I.e. without the `default=`.